2020-12-22 13:29:03|ERROR|User.WithdrawList|PDOException: Unknown column 'created_at' in 'field list' in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(811): NotORM_Result->query('SELECT change_m...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(896): NotORM_Result->execute()
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/User.php(959): NotORM_Result->fetchAll()
#3 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_User->withdrawList()
#4 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#5 {main}
2020-12-22 13:29:12|ERROR|User.WithdrawList|PDOException: Unknown column 'created_at' in 'field list' in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(811): NotORM_Result->query('SELECT change_m...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(896): NotORM_Result->execute()
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/User.php(959): NotORM_Result->fetchAll()
#3 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_User->withdrawList()
#4 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#5 {main}
2020-12-22 13:31:27|ERROR|User.WithdrawList|PDOException: Unknown column 'created_at' in 'field list' in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(811): NotORM_Result->query('SELECT change_m...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(896): NotORM_Result->execute()
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/User.php(959): NotORM_Result->fetchAll()
#3 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_User->withdrawList()
#4 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#5 {main}
2020-12-22 13:38:48|ERROR|User.Withdraw|PDOException: Unknown column 'type' in 'field list' in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(339): NotORM_Result->query('INSERT INTO cmf...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(356): NotORM_Result->insert_multi(Array)
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/User.php(914): NotORM_Result->insert(Array)
#3 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_User->withdraw()
#4 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#5 {main}
2020-12-22 13:40:30|ERROR|User.Withdraw|PDOException: Unknown column 'type' in 'field list' in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(339): NotORM_Result->query('INSERT INTO cmf...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(356): NotORM_Result->insert_multi(Array)
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/User.php(914): NotORM_Result->insert(Array)
#3 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_User->withdraw()
#4 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#5 {main}
2020-12-22 13:54:10|ERROR|User.WithdrawList|PDOException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ') LIMIT 1' at line 1 in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(811): NotORM_Result->query('SELECT bank_nam...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(887): NotORM_Result->execute()
#2 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(875): NotORM_Result->fetchRow('')
#3 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/User.php(965): NotORM_Result->fetchOne()
#4 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_User->withdrawList()
#5 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#6 {main}
2020-12-22 13:54:57|ERROR|User.WithdrawList|PDOException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ') LIMIT 1' at line 1 in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(811): NotORM_Result->query('SELECT bank_id,...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(887): NotORM_Result->execute()
#2 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(875): NotORM_Result->fetchRow('')
#3 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/User.php(964): NotORM_Result->fetchOne()
#4 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_User->withdrawList()
#5 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#6 {main}
